.github/workflows/maestro_tests.yml

```diff
@@ -74,8 +71,8 @@ jobs:
             
             # When installing Maestro manually, it doesn't get discovered in the path.
             # Hence, providing absolute path to binary.
-            $HOME/.maestro/bin/maestro test --format junit --output maestro-test-report.xml maestroUiFlows
-            
+            maestro test --format junit --output maestro-test-report.xml maestroUiFlows
+
```

Comment:
Can we also remove the comment about providing absolute path to binary, as now we have removed the absolute path?

Reply:
Good catch, fixing it.